src/main/kotlin/io/github/flank/gradle/tasks/FlankRunTask.kt

```diff
@@ -85,7 +90,9 @@ constructor(
           if (exitValue == NO_TESTS_EXIT_CODE) {
             logger.warn(
                 """
-                ${testApk.get().singleFile().asFile.relativeTo(projectLayout.projectDirectory.asFile)} doesn't contain any test or they are filtered out
+                ${
+              testApk.get().singleFile().asFile.relativeTo(projectLayout.projectDirectory.asFile)
+            } doesn't contain any test or they are filtered out
```

Comment:
the formatter did something weird here... 🙃  (and I guess it's better to make sure the changes are only the ones related to this PR)